### DISCUSSION TITLE
编译固件download dll库依赖时出错，提示Wrong hash

### DISCUSSION THREAD

**Discussion by a8223492** [Boos4721/openwrt#107] [Bugs/问题反馈]
下载时的提示如下 如何解决呢
time: package/utils/util-linux/download#0.09#0.02#0.14
remote: Total 387 (delta 0), reused 0 (delta 0), pack-reused 387
Receiving objects: 100% (387/387), 705.92 KiB | 470.00 KiB/s, done.
Resolving deltas: 100% (153/153), done.
Note: switching to 'f9ea82e2ab6c71b56374d160adfeaac7db291522'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at f9ea82e Upload board-2.bin from WLAN.HK.2.5.0.1-01201-QCAHKSWPL_SILICONZ-1  SI No modification done.
Packing checkout...
nss-firmware-2021-09-13-b06171cb.tar.xz: Download from https://github.com/quic/qca-sdk-nss-fw.git failed
nss-firmware-2021-09-13-b06171cb.tar.xz: Wrong hash (probably caused by .gitattributes), expecting b4638a25f1b4f63c91d57cad3d4b955fee7c796679cee55c557c2644db0f20e9, got 84e57df871eec6ce94c0241027c7f86b161fe5400d43f26bd93aa5a0ee8ad840
Checking out files from the git repository...
Cloning into 'nss-firmware-2021-09-13-b06171cb'...
remote: Enumerating objects: 59, done.
remote: Counting objects: 100% (59/59), done.
remote: Compressing objects: 100% (49/49), done.
make[2]: Leaving directory '/home/a8223492/Openwrt/BoosAX6/package/firmware/ath11k-board'
time: package/firmware/ath11k-board/download#5.39#0.16#8.43
remote: Total 59 (delta 15), reused 33 (delta 5), pack-reused 0
Unpacking objects: 100% (59/59), 2.63 MiB | 1.57 MiB/s, done.
Note: switching to 'b06171cb0d14360c90a4b8f9f835dc1c4647ea07'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at b06171c Add files via upload
Packing checkout...
make[2]: Leaving directory '/home/a8223492/Openwrt/BoosAX6/package/qca/firmware/nss-firmware'
time: package/qca/firmware/nss-firmware/download#1.28#0.14#9.10
ath11k-firmware-2022-03-04-4abc7856.tar.xz: Download from https://github.com/quic/upstream-wifi-fw.git failed
ath11k-firmware-2022-03-04-4abc7856.tar.xz: Wrong hash (probably caused by .gitattributes), expecting 7fa0f1d56e951a9351826710729312ca4d6a18866cb22d0cbaff49c628e75902, got 9f16dc1803c4caf67ff95b481770ac157e8a69da31fe39d36eb28deab167f6a3
Checking out files from the git repository...
Cloning into 'ath11k-firmware-2022-03-04-4abc7856'...
remote: Enumerating objects: 903, done.
remote: Counting objects: 100% (903/903), done.
remote: Compressing objects: 100% (669/669), done.
remote: Total 903 (delta 601), reused 466 (delta 169), pack-reused 0
Receiving objects: 100% (903/903), 47.10 MiB | 3.92 MiB/s, done.
Resolving deltas: 100% (601/601), done.
Note: switching to '4abc7856e26618a9b7a70f24d7f61e844f160e17'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 4abc785 IPQ5018_QCN6122_QCN6122 hw1.0: testing: 2.6.0.1.r7: add WLAN.HK.2.6.0.1.r7-00028-QCAHKSWPL_SILICONZ-1
Packing checkout...
make[2]: Leaving directory '/home/a8223492/Openwrt/BoosAX6/package/firmware/ath11k-firmware'
time: package/firmware/ath11k-firmware/download#92.68#3.29#133.24

**Comment by Boos4721** [comment]
![iShot2022-04-09_00 02 26](https://user-images.githubusercontent.com/43845654/162479719-ed807657-3476-4ea3-98b8-a6a5a62b1bf8.png)
U SHOULD CHECK UR NETWROK.
